.github/workflows/ci.yml

```diff
@@ -377,6 +381,14 @@ jobs:
             --secret=AWS_SECRET_ACCESS_KEY=${{ secrets.AWS_PSW }} \
             +ci-promote-build-artifacts --AWS_DEFAULT_REGION=us-east-1 --CROSSPLANE_VERSION=${CROSSPLANE_VERSION} --BUILD_DIR=${GITHUB_REF##*/} --CHANNEL=master
 
+      - name: Push Artifacts to https://releases.crossplane.io/preview/ and https://charts.crossplane.io/preview
+        if: env.AWS_USR != '' && github.ref == 'refs/heads/v2'
+        run: |
+          earthly --strict \
+            --secret=AWS_ACCESS_KEY_ID=${{ secrets.AWS_USR }} \
+            --secret=AWS_SECRET_ACCESS_KEY=${{ secrets.AWS_PSW }} \
+            +ci-promote-build-artifacts --AWS_DEFAULT_REGION=us-east-1 --CROSSPLANE_VERSION=${CROSSPLANE_VERSION} --BUILD_DIR=${GITHUB_REF##*/} --CHANNEL=preview
```

Comment:
This duplication isn't very elegant but given we expect the v2 branch to go away in a couple of months I'm not inclined to spend time figuring out a better way to do this.